nmostesting/NMOSTesting.py

```diff
@@ -540,12 +555,12 @@ def index_page():
 def run_tests(test, endpoints, test_selection=["all"]):
     if test in TEST_DEFINITIONS:
         test_def = TEST_DEFINITIONS[test]
-        protocol = "http"
-        if CONFIG.ENABLE_HTTPS:
-            protocol = "https"
+        protocol = "https" if CONFIG.ENABLE_HTTPS else "http"
         apis = {}
         tested_urls = []
         for index, spec in enumerate(test_def["specs"]):
+            if spec.get("websocket"):
+                protocol = "wss" if CONFIG.ENABLE_HTTPS else "ws"

```

Comment:
Doesn't this overwrite `protocol` for the next api in the loop which may be http?